Hangman/ViewController.m

```diff
@@ -11,6 +11,8 @@
 @interface ViewController ()
 @property (weak, nonatomic) IBOutlet UILabel *mysteryWordLabel;
 @property (weak, nonatomic) IBOutlet UITextField *userInputTextField;
+@property (weak, nonatomic) IBOutlet UILabel *guessCountLabel;
+@property int guessCounter;
```

Comment:
Keep in mind that once you start working with multiple ViewControllers you may need to access this variable outside of this class. If you do, you'll have to move this into your header file